src/btree/bt_io.c

```diff
@@ -35,16 +35,12 @@ __wt_bt_read(WT_SESSION_IMPL *session, WT_ITEM *buf, const uint8_t *addr, size_t
      * into the caller's buffer. Else, read directly into the caller's buffer.
      */
     if (btree->compressor == NULL && btree->kencryptor == NULL) {
-        WT_WITH_BUCKET_STORAGE(
-          btree->bstorage, session, { ret = bm->read(bm, session, buf, addr, addr_size); });
-        WT_RET(ret);
```

Comment:
Some explanation...  This was saying: use the bucket storage/file system in `btree->bstorage` when you do file system operations.  But the file handles are cached and opened lazily, so this single operation doesn't necessarily help.  We'd have to encapsulate every read and write into the block manager in these calls to make this work.  Far better to just catch this at the point of the lazy open.  Which is what we did -- WT_FILE_OPENER handles that.  So these macros don't really do anything (any effect they have to switch the file system is redone at the point of the lazy open).  Therefore they can all be removed (except at the lazy open).